src/components/dumb/OeHeader.vue

```diff
@@ -0,0 +1,201 @@
+              class="su0"
+            ></path>
+            <path d="M50.5-2.3h.4v87.7h-.4z" class="su0" transform="rotate(-18.785 50.698 41.547)"></path>
+            <path
+              d="M37 41.4c-2-1.6-3 0-4.3-.1-1.1-.1-2.1-1.8-2.9-1.4-1.6.8.6 3.7 1.6 4.3.9.5 2 1 2.2 1.1 1.3.6 1.8 1.5 2 2.9 0 .3 0 1.1-.1 1.4-.6 2.5-5 4.8-7.5 3-1.2-.8-2.3-1.9-2.6-3.7-.7-3.3-3-5.7-3.8-8.9-.5-1.9-.8-4-1.3-6s-1-4-1.5-5.9c-.4-1.7-1.2-4.3-1.8-5.8-2.4-6.3-3.5-5.8-3.5-5.8s.9 1.7 4 17.1c.1.6 1.3 7.4 2.1 9.6.3.8.8 2.5 1.2 3.2 1 2.2 3.7 5.5 3.8 8.5.1 1.7.3 3.2.3 4.5 0 .4.2 1.9.4 2.6.7 1.8 5.9 7.7 11.5 7.7v-4.5c-5.5 0-10.8-3.1-10.9-3.4-.1-.1.3-2.3.7-3.2.7-1.8 2.1-3.5 4.5-3.7 2.7-.2 4.3.6 5.7.6l.2-14.1zM12.9 25.1c-.2 3.4-5.4 7.9-7 10.9-.8 1.4-1.8 3.9-2 5.4-.7 4 .2 6.5 1.2 8.8 1.7 4.1-.2 5.6 1.1 4.8 1.7-1.4 1.4-4.5 1.3-6.5-.1-1.6-.2-3.4 0-5.3.5-3.6 2.6-7.4 4.5-10.1 2.1-3.2 1.3-6.5.9-8m1 9.6s.6 2.7-1.8 9.5C6 62 17.7 63.7 21.1 67.9c0 0 1.3-1.8-4-7.6-1.9-2.1-3.7-6.8-2.2-13.5 2.2-9.8-1-12.1-1-12.1M2.4 22.9c-.3-1.1-.4-2-.3-2.8.3-4.2 4.1-5.9 5-6.5 0 0 1.7-1.2 1.9-2.4 0 0 1.8 4.4-2.3 6.9-2 1.4-3.4 2.9-4.3 4.8m9.6-4c.2.4 1.8 2.7-4.7 8.1-6.5 5.3-4.4 9.1-4.4 9.1s-6.9-3.8 1-10.5c7.9-6.6 6.3-8 6.3-8s1.2 0 1.8 1.3m8.4 2c1.1.1 1.8 3.6 4.4 4.4 1.9.6 3.9.3 4.4 1.4-.9.5-.2 2 .7 1.7.7-2 1-9.8-9.5-7.5zm3.4 1.6c.1-.2.2 0 .5-.2.3-.3.6-.8 1.1-.9.4-.1.8-.1 1.2.1.2.1.1.6-.1.8-.3.3-1.4-.2-1.4.6 0 1.3 1.7 0 2.6 0 .5 3-4.6 2.1-3.9-.4z"
+              class="su0"
+            ></path>
+          </g>
+        </svg>
+      </a>
+
+      <h1 class="vl-u-spacer-left--small vl-u-spacer-right--small">
+        <vl-link to="/" exact>Plannen</vl-link>
```

Comment:
deze moeten configureerbaar zijn: appurl & appname ofzo

Reply:
Het blok met de naam van de app moet voor het blok met het logo komen, daar zitten we aan vast denk ik wegens huisstijl